app/views/verify/address/index.html.slim

```diff
@@ -1,5 +1,14 @@
-p Pick an address verification mechanism
+h1.h2
+  = t('idv.titles.select_verification')
+p
+  = t('idv.messages.select_verification', sp_name: @sp_name)
```

Comment:
I actually did not know that was possible. Should I have a fallback to APP_NAME if sp_name is nil, then?